PyInstaller/building/makespec.py

```diff
@@ -100,6 +100,11 @@ def __add_options(parser):
                         "(default: first script's basename)")
 
     g = parser.add_argument_group('What to bundle, where to search')
+    g.add_argument('--datas',
+                   action='append', default=[],
+                   metavar="FILE;DEST or FOLDER;DEST", dest='datas',
+                   help='Add an additional file or a folder to the executable. '
```

Comment:
I suggest something like  "Additional non-binary files or folders to be added to the executable. SRC may be a glob, see the manual for details."